_data/engine-cli/docker_service_update.yaml

```diff
@@ -38,7 +47,6 @@ options:
   default_value: '[]'
   description: Remove a DNS search domain
 - option: endpoint-mode
-  default_value: vip
```

Comment:
The default was added in 17.04 (this was considered safe to set as a default client-side); https://github.com/docker/docker/pull/30180